data/decks/web-full.yaml

```diff
@@ -0,0 +1,23 @@
+---
+name: Full Web test deck
+description: This deck runs HTTP Header Field Manipulation, HTTP Invalid
+    Request Line and the Web Connectivity test
+schedule: "@daily"
```

Comment:
Yeah `@` is a indicator character that according to the [spec](http://www.yaml.org/spec/1.2/spec.html#id2772075) is "reserved for future use".